contents/docs/product-analytics/_snippets/how-to-capture-identified-events-backend.mdx

```diff
@@ -0,0 +1,92 @@
+PostHog captures identified events by default. To capture identitfied events, either set the `$process_person_profile` property `true`, or simply omit this property when capturing events:
+
+<MultiLanguage>
+
+```node
+client.capture({
+    distinctId: 'distinct_id_of_the_user',
+    event: 'your_event_name',
+    properties: {
+        $process_person_profile: true, // or omit this property
+    },
+})
+```
+
+```python
+posthog.capture(
+    "distinct_id_of_the_user", 
+    "your_event_name", 
+    {
+        "$process_person_profile": true  # or omit this property
+    }
+)
+```
+
+```php
+PostHog::capture(array(
+  'distinctId' => 'distinct_id_of_the_user',
+  'event' => 'your_event_name',
+  'properties' => array(
+    '$process_person_profile' => true  # or omit this property
+  )
+));
+```
+
+```ruby
+posthog.capture({
+    distinct_id: 'distinct_id_of_the_user',
+    event: 'your_event_name',
+    properties: {
+        $process_person_profile: true  # or omit this property
+    }
+})
+```
+
+```go
+client.Enqueue(posthog.Capture{
+    DistinctId: "distinct_id_of_the_user",
+    Event:      "your_event_name",
+    Properties: posthog.NewProperties().
+      Set("$process_person_profile", true),  // or omit this property
+})
+```
+
+```java
+posthog.capture("distinct_id_of_the_user", "your_event_name", new HashMap<String, Object>() {
+  {
+    put("$process_person_profile", true);  // or omit this property
+  }
+});
+```
+
+```rust
+let mut event = Event::new("your_event_name", "distinct_id_of_the_user");
+
+event.insert_prop("$process_person_profile", true).unwrap();
+
+client.capture(event).unwrap();
+```
+
+```elixir
+Posthog.capture("your_event_name", %{
+  distinct_id: distinct_id_of_the_user,
+  properties: %{
+    "$process_person_profile": true  # or omit this property
+  }
+})
+```
+
+```API
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "properties": {
+        "$process_person_profile": true  # or omit this property
+    },
+    "distinct_id": "distinct_id_of_the_user",
+    "event": "your_event_name"
+}' <ph_client_api_host>/capture/
+```
```

Comment:
I think its more confusing having it there
```suggestion
```node
client.capture({
    distinctId: 'distinct_id_of_the_user',
    event: 'your_event_name',
})
```

```python
posthog.capture(
    "distinct_id_of_the_user", 
    "your_event_name"
)
```

```php
PostHog::capture(array(
  'distinctId' => 'distinct_id_of_the_user',
  'event' => 'your_event_name',
  'properties' => array(
    '$process_person_profile' => true  # or omit this property
  )
));
```

```ruby
posthog.capture({
    distinct_id: 'distinct_id_of_the_user',
    event: 'your_event_name'
})
```

```go
client.Enqueue(posthog.Capture{
    DistinctId: "distinct_id_of_the_user",
    Event:      "your_event_name",
)
```

```java
posthog.capture("distinct_id_of_the_user", "your_event_name");
```

```rust
let mut event = Event::new("your_event_name", "distinct_id_of_the_user");

client.capture(event).unwrap();
```

```elixir
Posthog.capture("your_event_name", %{
  distinct_id: distinct_id_of_the_user,
})
```

```API
curl -v -L --header "Content-Type: application/json" -d '{
    "api_key": "<ph_project_api_key>",
    "distinct_id": "distinct_id_of_the_user",
    "event": "your_event_name"
}' <ph_client_api_host>/capture/
```
```